src/components/profile/Recovery/RecoveryContainer.js

```diff
@@ -47,8 +48,15 @@ const RecoveryContainer = ({ type, recoveryMethods }) => {
 
     const handleDeleteMethod = async (method) => {
         try {
+            Mixpanel.track(method.kind === 'phrase'? 'SR-SP Delete method start': 'SR Delete method start')
             setDeletingMethod(method.publicKey)
-            await dispatch(deleteRecoveryMethod(method, deleteAllowed))
+            try {
+                await dispatch(deleteRecoveryMethod(method, deleteAllowed))
+                Mixpanel.track(method.kind === 'phrase'? 'SR-SP Delete method finish': 'SR Delete method finish')
```

Comment:
I really think it's worth including `method.kind` everywhere possible even if it deviates from your `SR-SP` pattern for seed phrase. Right now we only know what recovery method that was disabled if it was seed phrase.